src/main/resources/org/jenkinsci/plugins/stashNotifier/StashNotifier/global.jelly

```diff
@@ -25,5 +25,10 @@
                help="${rootURL}/plugin/stashNotifier/help-globalConfig-includeBuildNumberInKey.html">
           <f:checkbox />
       </f:entry>
+      <f:entry title="Prepend parent project key"
```

Comment:
Just as an idea to clarify: how about we rename the field label to "Prepend parent project name to key"?